algorithmic_efficiency/workloads/mnist/workload.py

```diff
@@ -44,6 +44,10 @@ def num_validation_examples(self):
   def num_test_examples(self):
     return 10000
 
+  @property
+  def eval_batch_size(self):
+    return 1024
```

Comment:
Here we could probably fit the whole eval datasets into memory?

Reply:
Fixed! I modified it to 10000.